pkg/instance/instance.go

```diff
@@ -506,7 +506,7 @@ var MakeBin = func() string {
 	return "make"
 }()
 
-func RunnerCmd(prog, fwdAddr, os, arch string, poolIdx, vmIdx int, collide, threaded, newEnv bool) string {
+func RunnerCmd(prog, fwdAddr, os, arch string, poolIdx, vmIdx int, threaded, newEnv bool) string {
```

Comment:
You asked how this can be split. This clearly looks like an unrelated change that can well go into a separate commit. It would remove 2/19 changed files in this PR.